<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>336</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>436</width>
    <height>336</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>436</width>
    <height>336</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_shadow" native="true">
     <widget class="QLabel" name="label_TopPicture">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>3</y>
        <width>431</width>
        <height>125</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>125</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>130</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_set">
      <property name="geometry">
       <rect>
        <x>346</x>
        <y>2</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Btn_close">
      <property name="geometry">
       <rect>
        <x>401</x>
        <y>2</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="widget_down" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>127</y>
        <width>430</width>
        <height>203</height>
       </rect>
      </property>
      <widget class="QPushButton" name="Btn_login">
       <property name="geometry">
        <rect>
         <x>111</x>
         <y>148</y>
         <width>221</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pwd">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_userName">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>44</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="statusTip">
        <string>QQ号码/手机/邮箱</string>
       </property>
       <item>
        <property name="text">
         <string>123</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>132</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2222</string>
        </property>
       </item>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>111</x>
         <y>113</y>
         <width>253</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_autoLogin">
          <property name="text">
           <string>自动登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_rememberPwd">
          <property name="text">
           <string>记住密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_retrievePwd">
          <property name="text">
           <string>找回密码</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_register">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>179</y>
         <width>60</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>注册账号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_QRCode">
       <property name="geometry">
        <rect>
         <x>392</x>
         <y>167</y>
         <width>25</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Btn_min">
      <property name="geometry">
       <rect>
        <x>373</x>
        <y>2</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
